src/ssa.rs

```diff
@@ -688,7 +691,7 @@ mod parse {
                 strikeout: parse_str_to_bool(
                     get_line_value(
                         &headers,
-                        "Strikeout",
+                        "StrikeOut",
```

Comment:
It needs to support `Strikeout` and `StrikeOut` i think. At least in the [spec](http://www.tcax.org/docs/ass-specs.htm) in the section 5 overview it says that the field is called `StrikeOut`, but in the more detailed explanation beneath (field 9.2) it says `Strikeout`

Reply:
In the get_line_value function it will be in lowercase anyway, so it does not matter anymore.